app/views/pages/sk-lang-privacy-policy.ejs

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
Oops. Please copy the format of the other privacy policy pages. The header and footer did change!